plugins/image-prioritizer/class-image-prioritizer-img-tag-visitor.php

```diff
@@ -126,4 +157,25 @@ static function ( string $value ): bool {
 
 		return true;
 	}
+
+	/**
+	 * Checks whether the given 'sizes' attribute includes the 'auto' keyword as the first item in the list.
+	 *
+	 * Per the HTML spec, if present it must be the first entry.
+	 *
+	 * @since n.e.x.t
+	 * @see auto_sizes_attribute_includes_valid_auto()
```

Comment:
```suggestion
```

Let's remove auto-sizes function as we also use WP future function `wp_sizes_attribute_includes_valid_auto` also